test/fast/nvm should remove the last trailing slash in $NVM_DIR

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -ex
+
+die () { echo "$@" ; exit 1; }
+
+export NVM_DIR_BASE="/tmp"
+export NVM_DIR="${NVM_DIR_BASE}/"
+
+\. ../../nvm.sh
+
+[ "${NVM_DIR}" = "${NVM_DIR_BASE}" ] || die 'nvm should remove the last trailing slash in "$NVM_DIR"'
+
+export NVM_DIR="${NVM_DIR_BASE}//"
+
+\. ../../nvm.sh
+
+[ "${NVM_DIR}" = "${NVM_DIR_BASE}" ] && die 'nvm should only remove the last trailing slash in "$NVM_DIR"'
```

Comment:
this "die" message seems incorrect; it should remove *all* trailing slashes in NVM_DIR, no?

Reply:
Nope, that was my original idea but rejected here: https://github.com/creationix/nvm/pull/1734#discussion_r168968027, I can revise it back if you want.

Reply:
ah - so what I intended in that comment was for nvm to *fail outright* when there was *any* trailing slashes in NVM_DIR.

The alternative is to remove all of them, and output a warning so they can fix it.

Reply:
yeah I've tested the case that there is only single slash and was removed, so to make sure we'll only remove the last slash, maybe I don't need to revise the tests here?

Reply:
To clarify further: if we go with removal, I think it should remove all trailing slashes, not just one.

Reply:
Rollbacked now ...